cpp/include/tensorrt_llm/batch_manager/kvCacheTransferManager.h

```diff
@@ -71,12 +98,23 @@ class KVCacheTransferManager
         int numTokensToCopy = 0, executor::KvCacheTransferMode mode = executor::KvCacheTransferMode::DRAM,
         std::string const& directory = "");
 
+    /*!
+     * \brief Directly copy a block from gpu to gpu.
```

Comment:
Is this asynchronous from CPU's perspective? If so, we should mention that here. Something like:

```
\note The copy is launched on an internal stream asynchronously and this function may return before the copy is complete.
```

Reply:
My approach has changed, a synchronize to mOnboardManager and mOffloadManager shall do. So the function is no longer needed.